README.md

```diff
@@ -2,7 +2,7 @@ SQLite JDBC Driver [![Build Status](https://travis-ci.org/xerial/sqlite-jdbc.svg
 ==================
 SQLite JDBC, developed by [Taro L. Saito](http://www.xerial.org/leo), is a library for accessing and creating [SQLite](http://sqlite.org) database files in Java.
 
-Our SQLiteJDBC library requires no configuration since native libraries for major OSs, including Windows, Mac OS X, Linux etc., are assembled into a single JAR (Java Archive) file. The usage is quite simple; [download](https://bitbucket.org/xerial/sqlite-jdbc/downloads)
+Our SQLiteJDBC library requires no configuration since native libraries for major OSs, including Windows, Mac OS X, Linux etc., are assembled into a single JAR (Java Archive) file. The usage is quite simple; [download](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22sqlite-jdbc%22%20g%3Aorg.xerial)
```

Comment:
Could you keep the URL as is? We are still using bitbucket for downloading.

Reply:
Will do